Update README.md

wannabeOG opened the pull request:
Changed it to a hyphen, you hyperventilating cuckoo

wannabeOG committed:
Update README.md

Changed it to a hyphen, you hyperventilating cuckoo

fristonio commented:
Remove spaces enclosed within [ x ] and add space in [] as in 
![image](https://user-images.githubusercontent.com/21292343/31966132-4ea13318-b927-11e7-93ed-8f6e272475d6.png)
to fix the checkbox.

fristonio commented:
@wannabeOG

wannabeOG committed:
Update README.md

Take this you fucking piece of shit 
Zindagi main tu aur main dono single marne vaalein
